README-MALAY.md

```diff
@@ -0,0 +1,142 @@
+- [Git-it](https://github.com/jlord/git-it-electron) - Langkah demi langkah untuk memuat turun aplikasi Git dalam desktop anda.
+- [Flight Rules for Git](https://github.com/k88hudson/git-flight-rules) - Panduan mengenai apa yang harus dilakukan apabila masalah anda tidak dapat diselesaikan.
+- [Git Guide for Beginners in Spanish](https://platzi.github.io/git-slides/#/) - Panduan lengkap mengenai slaid mengenai git dan GitHub dijelaskan dalam bahasa Sepanyol. Una guía completa de diapositivas sobre git y GitHub explicadas en Español.
+- [Git Kraken](https://www.gitkraken.com/git-client) - Aplikasi desktop `git` visual, lintas platform, dan interaktif untuk kawalan versi.
+- [Git Tips](https://github.com/git-tips/tips) - Koleksi petua dan trik git yang paling biasa digunakan.
+- [Git Best Practices](https://sethrobertson.github.io/GitBestPractices/) - Lakukan Selalunya, Sempurna Kemudian, Terbitkan Sekali: Git Amalan Terbaik
+- [Git Interactive Tutorial](https://learngitbranching.js.org/) - Belajar Git secara visual dan interaktif
+
+## Buku untuk Sumber Terbuka
+- [Producing Open Source Software](https://producingoss.com/) - Menghasilkan Perisian Sumber Terbuka adalah buku mengenai sisi manusia dalam pembangunan Sumber Terbuka. Ini menggambarkan bagaimana projek yang berjaya dijalankan, harapan pengguna dan pembangun, dan budaya perisian percuma.
+- [Open Source Book Series](https://opensource.com/resources/ebooks) - Belajar lebih mendalam mengenai Sumber Terbuka dan pergerakan Sumber Terbuka yang berkembang dengan senarai lengkap eBook percuma dari https://opensource.com.
+- [Software Release Practice HOWTO](https://en.tldp.org/HOWTO/Software-Release-Practice-HOWTO/) - Ini adalah bagaimana menerangkan amalan pelepasan yang baik untuk Linux dan projek Sumber Terbuka yang lain. Dengan mengikuti amalan ini, anda akan mempermudah pengguna membina kod anda dan menggunakannya, dan agar pembangun lain memahami kod anda dan bekerjasama dengan anda untuk memperbaikinya.
+- [Open Sources 2.0 : The Continuing Evolution](https://archive.org/details/opensources2.000diborich) (2005) - Sumber Terbuka 2.0 adalah koleksi karangan berwawasan dan pemikiran dari pemimpin teknologi masa kini yang terus melukis gambar evolusi yang berkembang dalam buku 1999, Sumber Terbuka: Suara dari Revolusi.
+- [The Architecture of Open Source Applications](http://www.aosabook.org/en/git.html) - Tunjukkan bagaimana pelbagai aspek Git berfungsi di bawah penutup untuk membolehkan aliran kerja diedarkan, dan bagaimana ia berbeza dengan sistem kawalan versi lain (VCS).
+- [Open Sources: Voices from the Open Source Revolution](https://www.oreilly.com/openbook/opensources/book/) - Esei dari perintis sumber terbuka seperti Linus Torvalds (Linux), Larry Wall (Perl), dan Richard Stallman (GNU).
+
+## Sumbangan inisiatif Sumber Terbuka
+- [Up For Grabs](https://up-for-grabs.net/) - Mengandungi projek untuk penyumbang yang baru.
```

Comment:
- change `Mengandungi projek untuk penyumbang yang baru` to `Mengandungi isu projek yang mesra penyumbang baru`.